src/midifunctor.cpp

```diff
@@ -313,6 +317,170 @@ FunctorCode InitTimemapTiesFunctor::VisitTie(Tie *tie)
     return FUNCTOR_SIBLINGS;
 }
 
+//----------------------------------------------------------------------------
+// InitTimemapNotesFunctor
+//----------------------------------------------------------------------------
+
+InitTimemapNotesFunctor::InitTimemapNotesFunctor() : Functor()
+{
+    m_noCue = false;
+    m_lastNote = NULL;
+    m_currentTempo = MIDI_TEMPO;
+}
+
+FunctorCode InitTimemapNotesFunctor::VisitChord(Chord *chord)
+{
+    if (chord->IsGraceNote()) {
+        std::list<Note *> notes;
+        const ListOfObjects &chordNotes = chord->GetList();
+        for (Object *obj : notes) {
+            Note *note = vrv_cast<Note *>(obj);
+            assert(note);
+            notes.push_back(note);
+        }
+
+        m_accentedGraceNote = (chord->GetGrace() == GRACE_acc);
+        double time = chord->HasGraceTime() ? chord->GetGraceTime() : 50.0;
+        const GraceGrp *graceGrp = vrv_cast<const GraceGrp *>(chord->GetFirstAncestor(GRACEGRP));
+        if (graceGrp) {
+            if (graceGrp->GetGrace() == GRACE_acc) m_accentedGraceNote = true;
+            time = (graceGrp->HasGraceTime()) ? graceGrp->GetGraceTime() : 50.0;
+        }
+        m_graces.push_back({ notes, chord->GetActualDur(), time });
+
+        return FUNCTOR_SIBLINGS;
+    }
+
+    return FUNCTOR_CONTINUE;
+}
+
+FunctorCode InitTimemapNotesFunctor::VisitGraceGrpEnd(GraceGrp *graceGrp)
+{
+    // Handling of Nachschlag
+    if (!m_graces.empty() && (graceGrp->GetAttach() == graceGrpLog_ATTACH_pre) && !m_accentedGraceNote && m_lastNote) {
+        Fraction startTime = m_lastNote->GetScoreTimeOffset();
+        const Fraction graceNoteDur = UNACC_GRANENOTE_FRACTION * (int)m_currentTempo / 120;
```

Comment:
The `graceNoteDur` here is `1/64 * 1/120 = 0,00013` while before it was `27 * 1/60000 = 0,00045 `. So unaccented grace notes appear more than three times shorter than before. This difference is audible, e.g. in the Nachschlag example in https://github.com/rism-digital/verovio/pull/2577 . Was this change on purpose and if yes, why?

Reply:
You are right. I'll change it to `1/2048`